app/decorators/skip_rule_decorator.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Generates human readable representation of Skip Rules
+class SkipRuleDecorator < ApplicationDecorator
+  delegate_all
+
+  def human_readable
+    I18n.t("skip_rule.instructions",
+      destination: display_dest,
+      conditions: decorate_conditions)
+  end
+
+  def decorate_conditions
```

Comment:
Do all the below methods need to be public? Default should be private, only public if they need to be called from outside.